bindings/generator/api/os.py

```diff
@@ -0,0 +1,21 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+from .common import Variant
+
+
+class OSes(Variant):
```

Comment:
`OS` ? Most of the time we don't call a variant plural (e.g. `UserProfile`, `HTTPError`)

Reply:
done